fellow-submissions/enmanuel-delanuez.md

```diff
@@ -95,3 +104,4 @@
 
     console.log(advice);
     ```
+    - This code 'works' because variable `advice` is hoisted and then assigned `Pack your umbrella.`
```

Comment:
This has less to do with hoisting and more to do with the scope of variables defined with `var`. We know that `let`/`const` variables are _block_ scoped. `var` variables are _function_ scoped.